### PR TITLE
Added Calibrated Equalised Odds Wrapper.

### PR DESCRIPTION
**Pull request by samurai-kant**
Evaluated it with COMPAS dataset giving acceptable results
image

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -1,4 +1,4 @@
-using Fairness
+using Fairness2
```
Comment (ashryaagr) [inline test/runtests.jl:1]: The name of the package should be Fairness and not Fairness2. This is causing the tests to fail

**Review by ashryaagr — CHANGES_REQUESTED** [review]
Also, could you add documentation (docstrings) for the functions/algorithm you have added, in the same fashion as we have for Equalized Odds, etc

File: test/runtests.jl (hunk 1 above)
Comment (samurai-kant) [reply test/runtests.jl]: Hi!
I Have changed the package name and it is working fine now.
Also added comments to the code.

**Comment by ashryaagr** [comment]
Please update the documentation as well for this algorithm [here](https://github.com/ashryaagr/Fairness.jl/blob/master/docs/src/algorithms.md) and add this algorithm to the [table](https://github.com/ashryaagr/Fairness.jl#fairness-algorithms) in the README.

**Review by ashryaagr — APPROVED** [review]

**Comment by ashryaagr** [comment]
Thanks a lot for the Pull Request! Calibrated Equalized Odds algorithm wrapper is an excellent contribution. Everything (docs, readme, tests, code, readability), looks perfect to me. 

I am merging this Pull Request. 

This algorithm's implementation is for binary protected attribute. It would be another great contribution to add multi-class support for this algorithm. Feel free to open a separate Pull Request for it!

**Comment by samurai-kant** [comment]
Thanks a lot! It really means a lot to me. I will try to add support for
mult-class as soon as possible.

**Pull request merged**